test/integration/initial-dns-seedlist-discovery/initial_dns_seedlist_discovery.prose.test.ts

```diff
@@ -0,0 +1,255 @@
+import { expect } from 'chai';
+import * as dns from 'dns';
+import * as sinon from 'sinon';
+
+import { MongoAPIError, Server, ServerDescription, Topology } from '../../mongodb';
+import { topologyWithPlaceholderClient } from '../../tools/utils';
+
+describe('Initial DNS Seedlist Discovery (Prose Tests)', () => {
+  context('1. When running validation on an SRV string before DNS resolution', function () {
```

Comment:
For easy audit purposes (e.g. in evg), we should match the number and title of the prose test in the context block and capture the full instructions text in comments.